server/user.go

```diff
@@ -82,6 +82,24 @@ func httpUserConnect(ji Instance, w http.ResponseWriter, r *http.Request) (int,
 	return http.StatusFound, nil
 }
 
+func httpUserToDocsorConnect(ji Instance, w http.ResponseWriter, r *http.Request) (int, error) {
+	mattermostUserID := r.Header.Get("Mattermost-User-Id")
+	if mattermostUserID == "" {
+		return http.StatusUnauthorized, errors.New("not authorized")
+	}
+
+	// If user is already connected we show them the docs
+	jiraUser, err := ji.GetPlugin().userStore.LoadJIRAUser(ji, mattermostUserID)
+	if err == nil && len(jiraUser.Key()) != 0 {
+		jiraPluginDocsLink := "https://github.com/mattermost/mattermost-plugin-jira#1-features"
```

Comment:
We are migrating the docs from the README to GitBooks. Though we need to decide when we are making the switch entirely to avoid these in-between interactions like this. cc @aaronrothschild

Reply:
Hello @mickmister , thanks for the review. What should I change this to at the moment?

Reply:
I think this is fine :+1: